ci-operator/config/openshift-psap/ci-artifacts/openshift-psap-ci-artifacts-master.yaml

```diff
@@ -34,7 +34,7 @@ tests:
   steps:
     test:
     - as: presubmit-operatorhub
-      commands: run gpu-operator test_operatorhub
+      commands: run nightly gpu-operator test_operatorhub
```

Comment:
Kinda weird that it's called nightly since it's not a periodic job

Reply:
it's a presubmit test *running the nightly test scenario*

Reply:
I know that, it's weird / misleading
But I guess it's too late

Reply:
what would you suggest?

Reply:
s/nightly/(run|test|flow)/g 

`nightly` is not ideal to describe a the run/test/flow's name, it's more fitting to describe the CI job that runs said run/test/flow

Reply:
changed to `prow` as this directory is PROW CI specific.

Reply:
https://github.com/openshift-psap/ci-artifacts/pull/261

I'll update this PR when https://github.com/openshift-psap/ci-artifacts/pull/261 is merged to avoid useless rehearsals